app/jobs/send_sms_job.rb

```diff
@@ -1,18 +1,26 @@
 class SendSmsJob < ApplicationJob
   queue_as :default
 
-  def perform(date, details)      
-
+  def perform(date, details)
     @date = Date.parse(date)
     @details = details
     @converted_date = @date.strftime('%d-%B-%Y')
+    facility_id = User.current.location_id
+    
+    sms_gateway_url = get_global_property("#{facility_id}_sms_gateway_url")
```

Comment:
Let us not keep api_key and sms gateway in the database let still maintain it in the config file application.yml let us only keep the messages we send in the db.